plugins/plugin-chart-echarts/src/MixedTimeseries/EchartsMixedTimeseries.tsx

```diff
@@ -16,10 +16,92 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React from 'react';
-import { EchartsProps } from '../types';
+import React, { useCallback } from 'react';
+import { EchartsMixedTimeseriesChartTransformedProps } from './types';
 import Echart from '../components/Echart';
+import { EventHandlers } from '../types';
 
-export default function EchartsMixedTimeseries({ height, width, echartOptions }: EchartsProps) {
-  return <Echart height={height} width={width} echartOptions={echartOptions} />;
+export default function EchartsMixedTimeseries({
+  height,
+  width,
+  echartOptions,
+  setDataMask,
+  labelMap,
+  labelMapB,
+  groupby,
+  groupbyB,
+  selectedValues,
+  formData,
+  seriesBreakdown,
+}: EchartsMixedTimeseriesChartTransformedProps) {
+  const isFirstQuery = useCallback((seriesIndex: number) => seriesIndex < seriesBreakdown, [
```

Comment:
Here we need to determine which rawSeriesData was clicked (rawSeriesA or rawSeriesB)